activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Add logging to report async queries lock wait duration
+
+    Active Record logs will now display the time it took to complete an async query,
+    this timing is useful to know how efficient the query is.
+
+    ```
+    DEBUG -- :   Category Load (62.1ms)  SELECT *, sleep(0.001) FROM `categories` LIMIT 50
+    DEBUG -- :   ASYNC (64ms) Post Load (126.1ms)  SELECT *, sleep(0.001) FROM `posts` LIMIT 100
```

Comment:
Maybe cleanup the query to not include the `sleep`? I use this to generate slowish queries without having to insert loads of test data, but it makes for very weird example.

Also that's no longer what the log format look like, and it would be worth explaining what both timing means.